OwlRabbitFoxMerge.cs

Hassan0Kamel opened the pull request:
basically farms all the items in the 3 merge shops in /rivensylth I found it's better to merge them in one script since the farming quest is the same and everything look identical

Hassan0Kamel committed:
OwlRabbitFoxMerge.cs

basically farms all the items in the 3 merge shops in /rivensylth I found it's better to merge them in one script since the farming quest is the same and everything look identical

wtffidy commented on Other/MergeShops/OwlRabbitFoxMerge.cs:
```diff
@@ -0,0 +1,94 @@
+                case "Stories of the Forest":
+                    Core.FarmingLogger(req.Name, quant);
+                    Core.RegisterQuests(4645);
+                        Core.HuntMonster("rivensylth", "Rivensylth Spider", req.Name, quant, true, false);
```
you'll need the while line here

while (!Bot.ShouldExit && !Core.CheckInventory(req.Name, quant))
{
huntline with the req.name as the req for the quest
}


since the reward comes from a quest?
if the item is strictly from a mob u don't need the registerquest

Hassan0Kamel committed:
Merge branch 'BrenoHenrike:Skua' into Skua

Hassan0Kamel committed:
Owl, Rabbit and Fox [MEM] Merge shops

farms the items from owl merge, rabbit merge and fox merge in /rivensylth , not cool items but ye

Hassan0Kamel commented:
separated the scripts and made some changes
